bin/drush-install.sh

```diff
@@ -35,6 +35,31 @@ org.civicrm.contactsummary,\
 org.civicrm.bootstrapcivicrm,\
 org.civicrm.bootstrapcivihr
 
+## List of Default Enabled Cron Jobs
+ENABLED_JOBS=( "CiviCRM Update Check" \
+"Clean-up Temporary Data and Files" \
+"Clone Documents" \
+"Disable expired relationships" \
+"Length of service updater" \
+"Mail Reports" \
+"Process Inbound Emails" \
+"Send Scheduled Mailings" \
+"Send Scheduled Reminders" \
+"Tasks and Assignments Daily Reminder" \
+"Documents Notification" \
```

Comment:
For Jobs created by CiviCRM, I think this approach is good, but for Jobs created by extensions (e.g., Tasks and Assignments Daily Reminder and Lenght of Service updater), I'm not sure it this is the best way to do it.

For example, the Tasks and Assignments Daily Reminder probably uses code from the T&A extension (classes, BAOs etc). If the extension gets disabled the job will probably fail (because it will try to access an API that doesn't exist anymore). I feel it's better to keep the logic to enable and disable the scheduled job inside the extension and use hooks to disable/enable it when the extensions gets disabled/enabled (and remove it when the extension is removed). @omarabuhussein what do you think?